src/MgvGovernable.sol

```diff
@@ -17,6 +17,7 @@ contract MgvGovernable is MgvRoot {
       /* Set initial gasprice and gasmax. */
       setGasprice(_gasprice);
       setGasmax(gasmax);
+      setMaxDepth(40);
```

Comment:
Two comments:

1. Perhaps we should name this parameter in a more semantic way. Eg `maxOfferTakeCount`, `maxTakenOffers` or smth. Or perhaps we should say what 'depth' this is referring to.

Reply:
I would say `maxRecursionDepth`

Reply:
The change has been made.